src/pages/merge/MergeIndividuals.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import { useLocation } from 'react-router';
+import { get } from 'lodash-es';
+
+import Grid from '@material-ui/core/Grid';
+
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import MainColumn from '../../components/MainColumn';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import useIndividual from '../../models/individual/useIndividual';
+import useMergeIndividuals from '../../models/individual/useMergeIndividuals';
+import useGetMergeConflicts from '../../models/individual/useGetMergeConflicts';
+import {
+  derivePropertyOverrides,
+  isFormComplete,
+} from './utils/formDataUtils';
+import ResolutionSelector from './ResolutionSelector';
+import IndividualCard from './IndividualCard';
+
+export default function MergeIndividuals() {
+  const { search } = useLocation();
+  useDocumentTitle('MERGE_INDIVIDUALS');
+
+  const searchParams = new URLSearchParams(search);
+  const individualIds = searchParams.getAll('i') || [];
+
+  const { data: mergeConflicts } = useGetMergeConflicts(
```

Comment:
Might not be a bad idea to handle errors and loading associated with getMargeConflicts, useIndividual (1 & 2)?